validate/validators/bam.py

```diff
@@ -27,7 +28,8 @@ def check_bam_index(path):
 
     return True
 
-def check_bam(path:Path):
+# pylint: disable=W0613
+def check_bam(path:Path, args:Union[argparse.Namespace,None]):
```

Comment:
For nullable types (types that can be `None`), use `Optional[T]` - this would look like `args: Optional[argparse.Namespace] = None`